docs/_includes/engage.html

```diff
@@ -94,30 +94,30 @@ <h4 class="section-subheading">{{ site.data.sitetext.engage.subtext }}</h4>
               </div>
             </div>  
           </div>
-          <!-- Meeting Minutes -->
+          <!-- CNCF Community -->
           <div class="col-lg-4">
             <div class="card-web card hover-up hover-line"> 
               <div class="card-body">
-                <a href="https://bit.ly/3B7uIy9"><h4 class="card-title"><i class="far fa-file cmu"></i> Meeting Minutes</h4>
+                <a href="https://community.cncf.io/service-mesh-performance-community/"><h4 class="card-title"><i class="far fa-comments cmu"></i> Cloud Native Community Group</h4>
                 <p class="card-text">
-                  Have a topic to discuss? Add it to the meeting minutes.
+                  Join the SMP Cloud Native Community Group!
                 </p>
                 </a>
               </div>
             </div>  
           </div>
-          <!-- CNCF Community -->
+          <!-- Meeting Minutes -->
           <div class="col-lg-4">
             <div class="card-web card hover-up hover-line"> 
               <div class="card-body">
-                <a href="https://community.cncf.io/service-mesh-performance-community/"><h4 class="card-title"><i class="far fa-comments cmu"></i> Cloud Native Community Group</h4>
+                <a href="https://bit.ly/3B7uIy9"><h4 class="card-title"><i class="far fa-file cmu"></i> Meeting Minutes</h4>
                 <p class="card-text">
-                  Join the SMP Cloud Native Community Group!
+                  Join the Service Mesh Performance Cloud Native Community Group!
```

Comment:
```suggestion
                 Have a topic to discuss? Add it to the meeting minutes.
```

This line will be this only, sorry my bad i should've been more clear.

Reply:
line 115 has to this. while line 103 is the change that i had asked for